setup.py

```diff
@@ -10,21 +10,13 @@
 from setuptools.command.develop import develop
 from setuptools.command.install import install
 
-try:
-    from pip._internal.req import parse_requirements
-except:
-    from pip.req import parse_requirements
-
 
 if __name__ == '__main__':
     requirements_path = 'requirements.txt'
     if sys.version_info[0] < 3:
         requirements_path = 'requirements_py2.txt'
-    install_reqs = parse_requirements(requirements_path, session=False)
-    try:
-        reqs = [str(ir.req) for ir in install_reqs]
-    except:
-        reqs = [str(ir.requirement) for ir in install_reqs]
+    with open(requirements_path) as f:
```

Comment:
This assumes you're running from the location of the requirements txt, is this always true?

Reply:
i think the old `parse_requirements(requirements_path, session=False)` already made that assumption, so that should be fine.